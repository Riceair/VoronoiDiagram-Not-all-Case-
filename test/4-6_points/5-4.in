5
280 189
22 343
241 387
406 487
502 236