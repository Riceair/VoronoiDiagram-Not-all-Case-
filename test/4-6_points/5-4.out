P 22 343
P 241 387
P 280 189
P 406 487
P 502 236
E -96868.71 -162273.5 151.29 266.5
E -90506 242335 14.65 946.61
E 14.65 946.61 151.29 266.5
E 14.65 946.61 385.2 335.19
E 151.29 266.5 370.44 309.66
E 370.44 309.66 385.2 335.19
E 370.44 309.66 30001 -139647.5
E 385.2 335.19 158584 60841.5
